### DISCUSSION TITLE
Linux kali 2021.4a  运行wifipumpkin3 报错

### DISCUSSION THREAD

**Discussion by szhyt** [P0cL4bs/wifipumpkin3#173] [Q&A] [answered]
File "/usr/local/lib/python3.9/dist-packages/wifipumpkin3-1.0.9-py3.9.egg/wifipumpkin3/core/wirelessmode/wirelessmode.py", line 150, in get_error_hostapdServices
    raise HostapdInitializeError(
wifipumpkin3.exceptions.errors.hostapdException.HostapdInitializeError: [ERROR] Hostpad Failed: check output process hostapd.
 Configuration file: /root/.config/wifipumpkin3/config/hostapd/hostapd.conf
Line 4: invalid bssid item
1 errors found in configuration file '/root/.config/wifipumpkin3/config/hostapd/hostapd.conf'
Failed to set up interface with /root/.config/wifipumpkin3/config/hostapd/hostapd.conf
Failed to initialize interface
zsh: abort      wifipumpkin3

**Comment by mh4x0f** [comment, marked as answer]
the bssid is insvalid you need to set a valid bssid
